### DISCUSSION TITLE
Emulate Activities for real time test

### DISCUSSION THREAD

**Discussion by Abacaxi-Nelson** [GetStream/stream-feed-flutter#180] [Q&A]
Bonjour @sachaarbonel 

Im developing realtime notification on my feed when new activities are created.
Using same code you provide :
 ```dart
subscription = await userFeed.subscribe((message) {
...
}
```

I would like to know if its possible to trigger some **RealtimeMessage message** for test purpose ?
Creating test activities is as well possible, but as now, we only have one env, and not easy to always remove test activities..

Merci,
Nelson

**Comment by sachaarbonel** [comment]
Hey sorry for the late response @Abacaxi-Nelson, didn't see the discussions. 
Hmm, do you think a [cli](https://github.com/GroovinChip/stream_feed_cli) would be a good fit in your workflow? I don't think @GroovinChip implemented it already but if you think that would help let us know
